homeassistant/components/sensor/recorder.py

```diff
@@ -0,0 +1,96 @@
+"""Statistics helper for sensor."""
+from __future__ import annotations
+
+import datetime
+import logging
+import statistics
+
+from homeassistant.components import history
+from homeassistant.const import ATTR_DEVICE_CLASS
+from homeassistant.core import HomeAssistant
+from homeassistant.util.async_ import run_callback_threadsafe
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+DEVICE_CLASSES = {"temperature"}
+DEVICE_CLASS_STATISTICS = {"temperature": {"mean", "min", "max"}}
+
+
+def _get_entities(hass: HomeAssistant) -> list[tuple[str, str]]:
+    """Get (entity_id, device_class) of all sensors for which to compile statistics."""
+    all_sensors = hass.states.async_entity_ids(DOMAIN)
```

Comment:
```suggestion
    all_sensors = hass.states.all(DOMAIN)
```

Reply:
Fixed in 11542984aa06bd43dfe49112bbf1d92e6b51297e